CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,36 @@
+# Code of Conduct
+
+## Intro
+
+Whisky Oriented Development is dedicated to providing a safe environment for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. We do not tolerate harassment of members in any form. Members violating these rules may be sanctioned or expelled from the community or events (without a refund) at the discretion of the organizers.
+
+Harassment includes, but is not limited to:
+
+- Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following 
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behaviour 
+
+## Alcohol
+
+Whisky Oriented Development requires that all participants drink responsibly. This means not showing up to events intoxicated, not over consuming at events, and never driving after consuming alcohol.
+
+## Enforcement
```

Comment:
May want to add some language around "without refund" if an enforcement action has to occur.

Reply:
Good call.  How'd you find this PR anyways?

Reply:
I've been subscribed to this repo for years.